build/pkgs/flint/spkg-configure.m4

```diff
@@ -1,6 +1,6 @@
 SAGE_SPKG_CONFIGURE([flint], [
     SAGE_SPKG_DEPCHECK([mpfr], [
-        AC_CHECK_HEADER(flint/flint.h, [dnl
+        AC_CHECK_HEADERS([flint/flint.h flint/padic.h], [dnl
           dnl gr_get_fexpr appears in Flint 3.0
```

Comment:
Do you want to check `padic_types.h` or `padic.h`?

Reply:
intended as done here

Reply:
On my system:
```
flint$ ls -l padic*
-rw-r--r--  1 kwankyu  admin   8583 Feb 26 01:47 padic.h
-rw-r--r--  1 kwankyu  admin   6365 Feb 26 01:47 padic_mat.h
-rw-r--r--  1 kwankyu  admin  11871 Feb 26 01:47 padic_poly.h
flint$ pwd
/usr/local/Cellar/flint/3.1.0/include/flint
```
So `padic_types.h` is the missing one. No?

Reply:
checking for header files that the Sage library uses directly.

Reply:
I see. `AC_CHECK_HEADERS` checks if `flint/padic.h` works correctly. OK.

Reply:
That's right, it does so by compiling a test program.